### PR TITLE
Fix dep package describe and ci words spell.

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change

As now go mod is default package manager, delete the old dep package.
image

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by yanggangtony** [comment]
image

There is still one codespell error , but i can not found the files...

**Comment by blackpiglet** [comment]
https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/v1.5/restic.md

**Comment by qiuming-best** [comment]
> image
> There is still one codespell error , but i can not found the files...
> 
> image
> Is anybody can tell me , where is the `site/content/docs/v1.5/restic.md`?

I could find it [here](https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/v1.5/restic.md)

**Comment by yanggangtony** [comment]
@blackpiglet 
Thanks , i will search and update [v1.5/restic.md](https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/v1.5/restic.md).

File: pkg/controller/backup_controller.go [hunk 1]
```diff
@@ -540,17 +540,17 @@ func (b *backupReconciler) validateAndGetSnapshotLocations(backup *velerov1api.B
 	if len(errors) > 0 {
 		return nil, errors
 	}
-	allLocations := &velerov1api.VolumeSnapshotLocationList{}
-	err := b.kbClient.List(context.Background(), allLocations, &kbclient.ListOptions{Namespace: backup.Namespace, LabelSelector: labels.Everything()})
+	allocations := &velerov1api.VolumeSnapshotLocationList{}
```
Comment (blackpiglet) [inline pkg/controller/backup_controller.go:543]: `allocations` does not seem right here. `allLocations` refers to all the VolumeSnapshotLocations found in the Velero namespace.
Please use some wording like `volumeSnapshotLocations`, `vsls`, or `locations` instead.

**Comment by blackpiglet** [comment]
@yanggangtony 
I don't think we should delete all vendor dependencies from documents.
It looks like the vendor dependencies are used until v1.4.x, so the vendor dependencies should be kept for v1.3.x and older versions.

File: pkg/controller/backup_controller.go (hunk 1 above)
Comment (yanggangtony) [reply pkg/controller/backup_controller.go:543]: Got it.

**Comment by yanggangtony** [comment]
> @yanggangtony I don't think we should delete all vendor dependencies from documents. It looks like the vendor dependencies are used until v1.4.x, so the vendor dependencies should be kept for v1.3.x and older versions.

OK , i will revert it as your suggestion.

**Review by blackpiglet — APPROVED** [review]

**Review by anshulahuja98 — APPROVED** [review]

**Pull request merged**